content/docs/introduction/billing.md

```diff
@@ -0,0 +1,199 @@
+
+      ```text
+      main   ---------########>
+                      ^        
+                  snapshot
+
+      Legend:
+
+      ####### point-in-time-restore window, which is 
+              retained data history in the form of WAL 
+              records
+
+      ------- data history (WAL) that has fallen out of the 
+              point-in-time-restore window, and can no
+              longer be accessed
+      ```
+
+  - A _database branch_ is a snapshot of your data (including the parent branch's retained history) at the point of branch creation combined with WAL records that capture data changes from that point forward.
+
+      When a branch is first created, it adds no storage. No data changes have been introduced yet and the branch's snapshot data still exists in the parent branch's point-in-time restore window, which means that it shares this data in common with the parent branch.
+
+      ```text
+      main   ---------########>
+                      ^      |
+                   snapshot  |
+                             |
+      branch A               #>
+                             ^
+                          snapshot  
+      ```
+
+      A branch only begins adding to storage when a) data changes are introduced:
+
+      ```text
+      main   -------------#######>
+                          ^   |
+                   snapshot   |
+                              |
+      branch A                ####>
+                              ^
+                           snapshot  
+      ```
+
+      and b) when the branch snapshot falls out of the parent branch's point-in-time-restore window, in which case the branch snapshot data is no longer shared in common with the parent branch.
+
+      ```text
+      main   --------------------#######>
+                              |  ^
+                              |  snapshot
+                              |
+      branch A                ##########>
+                              ^
+                           snapshot
+      ```  
+
+      In other words, branches add storage when you modify data and when you allow the branch to age out of the parent branch's point-in-time-restore window.
+
+      Database branches can also share data history. For example, two branches created from the same parent at or around the same time will share data history, which avoids additional storage. The same holds true for a branch created from another branch. Wherever possible, Neon minimizes the storage cost of branches through shared data history. If it helps keeps storage size to a minimum, Neon will also take a new a branch snapshot to reduce the amount data changes stored as WAL.
+
+The cost calculation for _Project storage_ is as follows:
+
+```text
+project storage (GiB) * (seconds stored / 60) * price per hour
```

Comment:
should it be `(seconds stored / 3600) * price per hour` instead?

Reply:
@acervantes23 Can you comment on the question above?

Reply:
Good catch @lubennikovaav, let's go with: `(seconds stored / 3600) * price per hour`

Reply:
changed. Thank you.